projects/fsastorefrontlib/src/cms-components/my-account/quote/quote-details/quote-details.component.ts

```diff
@@ -40,11 +41,13 @@ export class QuoteDetailsComponent implements OnInit, OnDestroy {
       ])
         .pipe(
           map(([routingData, userId]) => {
-            const quoteId = routingData.state.params.quoteId;
-            if (quoteId) {
-              this.quoteService.loadQuoteDetails(quoteId);
+            if (!routingData.nextState) {
```

Comment:
This additional condition is added because `getRouterState ` method emits value even when the user clicks on the back icon. That causes unnecessary http call to the server.